### PR TITLE
Feat: Buttons added

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+.btn-primary{
+    color: var(--background-color);
+    background-color: var(--primary-color);
+    border:0.125rem solid var(--primary-color);
+}
+.btn-secondary{
+    color: var(--secondary-color);
+    background-color: var(--background-color);
+    border:0.125em solid var(--secondary-color);
```
Comment (Parul1999) [inline components/button/button.css]: Same units need to be used(either rem or em)

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+.btn-primary:hover:enabled{
+    box-shadow:  4px 4px 6px 0 rgba(255,255,255,.5),
+              -4px -4px 6px 0 rgba(216, 227, 240, 0.5), 
+    inset -4px -4px 6px 0 rgba(255,255,255,.2),
+    inset 4px 4px 6px 0 rgba(231, 243, 247, 0.4);
+    border-radius: 4px;
```
Comment (Parul1999) [inline components/button/button.css]: Use px or rem or em

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+.btn-secondary:hover:enabled{
+    color: #fff;
```
Comment (Parul1999) [inline components/button/button.css]: Use css variables

**Review by Parul1999 — CHANGES_REQUESTED** [review]
Use same units e.g border having different units at different places.

File: components/button/button.css (hunk 1 above)
Comment (Rohanmond) [reply components/button/button.css]: thanks for poinint out! it is fixed now

File: components/button/button.css (hunk 2 above)
Comment (Rohanmond) [reply components/button/button.css]: it is also fixed now

**Review by ravikumar1002 — APPROVED** [review]
Parul already pointed out  what changes was needed, after that everything is fine

File: components/button/button.css (hunk 3 above)
Comment (Rohanmond) [reply components/button/button.css]: Thanks for your comment. It is fixed now

**Pull request merged**